docs/resources/trigger.md

```diff
@@ -8,7 +8,7 @@ description: |-
 
 # wallarm_trigger
 
-Provides the resource to manage triggers. Triggers are tools that are used to set up custom notifications and reactions to events. Using triggers, you can receive alerts on major events via the tools you use for your day-to-day workflow, for example via corporate messengers or incident management systems.
+Provides the resource to manage [triggers][1]. Triggers are tools that are used to set up custom notifications and reactions to events. Using triggers, you can receive alerts on major events via the tools you use for your day-to-day workflow, for example via corporate messengers or incident management systems.
 
 To reduce the amount of noise, you can also configure the parameters of events to be notified about. The following events are available for setup:
```

Comment:
there are much more events than listed below. does provider support only the listed ones?

Reply:
@aoleynikov-wallarm, are only listed conditions supported?

Reply:
It supports one additional event: bruteforce started.